instrumentation/othttp/handler.go

```diff
@@ -79,6 +89,21 @@ func (h *Handler) configure(c *Config) {
 	h.spanNameFormatter = c.SpanNameFormatter
 }
 
+func (h *Handler) createMeasures() {
+	h.counters = make(map[string]*metric.Int64Counter)
+	h.valueRecorders = make(map[string]*metric.Int64ValueRecorder)
+
+	requestCounter := h.meter.NewInt64Counter(RequestCount)
+	requestBytesCounter := h.meter.NewInt64Counter(RequestBytes)
+	responseBytesCounter := h.meter.NewInt64Counter(ResponseBytes)
+	serverLatencyMeasure := h.meter.NewInt64ValueRecorder(ServerLatency)
```

Comment:
This could use a `metric.WithUnit()` annotation. I wonder if the service should expose microseconds, not millisecond units?

Reply:
I personally think microseconds is a better base unit - especially if you're using an `Int64Value`.  1500 µs is better than either 1 or 2 ms, depending on rounding.

Reply:
fixed